bedrock/privacy/templates/privacy/base-protocol.html

```diff
@@ -30,23 +28,17 @@
 {% endif %}
 
 {% block content %}
-<article class="section-content mzp-t-firefox" itemtype="http://schema.org/Article">
-  <header class="privacy-head">
-    <div class="l-narrow">
-      <div class="policy-title">
-        <h1 itemprop="name">{{ header.h2|join|safe }}</h1>
-        <time datetime="{{ header.select('[datetime]')[0]['datetime'] }}" itemprop="dateModified">{{ header.select('[datetime]')[0].string }}</time>
-      </div>
+<article class="section-content mzp-l-content mzp-t-firefox" itemtype="http://schema.org/Article">
+  <div class="l-narrow">
+    <div class="policy-title">
```

Comment:
Nit: should we keep this as a header element?

Reply:
Added the headers back. Thanks!